tools/iam-recommender-at-scale/README.md

```diff
@@ -0,0 +1,305 @@
+entire project using a set of commands in Cloud Shell.
+
+Here is that process in four steps (pre-step not included):
+
+1.  View the total number of service accounts, users, and groups that have IAM
+    Recommendations broken out by project.
+2.  Identify a project with some IAM recommendations that you feel comfortable
+    testing out the IAM Recommender on. We used a sandbox project for this step,
+    but you can do whatever seems best for you.
+3.  Bulk-apply recommendations on that project.We’ll also show you how to revert
+    this bulk-application as well.
+4.  Identify the next project that has recommendations and apply those too.
+
+## Pre-step
+
+Before step 1, there is a bit of work that needs to be done to make sure you and
+your GCP environment are ready:
+
+-   Check and make sure that you have the
+    [Recommender API](https://cloud.google.com/recommender/docs/reference/rest)
+    enabled.
+-   [Create a Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+    and give it the
+    [IAM Recommender Viewer](https://cloud.google.com/iam/docs/understanding-roles#recommender-roles),
+    [IAM Role Viewer](https://cloud.google.com/iam/docs/understanding-roles#roles-roles),
+    and
+    [Cloud Asset Viewer role](https://cloud.google.com/iam/docs/understanding-roles#cloud-asset-roles).
+    You will need to be able to reference this Service Account and its
+    associated key later while running this script.
+
+## STEP-1: viewing total number of IAM Recommendations
+
+1.  Run the below command in your [Cloud Shell](https://cloud.google.com/shell)
+    to save all the required code in a folder named `iam_recommender_at_scale`.
+    This command will also create a python virtual environment within the folder
+    to execute the code.
+
+    ```
+    curl https://raw.githubusercontent.com/GoogleCloudPlatform/professional-services/master/tools/iam-recommender-at-scale/install.sh | bash -
+    ```
+
+2.  Enter the below command to go to the source directory and activate the
+    python environment.
+
+    ```
+    cd iam-recommender-at-scale
+    . env/bin/activate
+    ```
+
+3.  Next, run the below code to retrieve all IAM Recommendations in your
+    organization and break it out by project. Make sure to enter in your
+    Organization ID, called out here as,`<YOUR-ORGANIZATION-ID>`. You’ll also
+    need to include a path to the service account key you created earlier,
+    called out below as, `<SERVICE-ACCOUNT-FILE-PATH>`.
+
+    ```
+    python get_projects_security_status.py \
+    --organization=<organizations/YOUR-ORGANIZATION-ID> \
+    --service_account_file_path=<SERVICE-ACCOUNT-FILE-PATH> \
+    --to_csv=<PATH-TO-CSV-FILE (optional)>
+    ```
+
+4.  For this demo we exported the results from step 1.2 into a CSV and uploaded
+    that CSV into a Google Sheet.
+
+    ![project security status](./asset/project-security-status.png)
```

Comment:
This image path doesn't render. Perhaps remove the leading `./`

Reply:
fixed.